core/src/test/java/hudson/util/io/RewindableRotatingFileOutputStreamTest.java

```diff
@@ -9,16 +9,20 @@
 import java.io.File;
 import java.io.IOException;
 import java.io.PrintWriter;
+import org.apache.commons.io.FileUtils;
+import org.junit.Rule;
+import org.junit.rules.TemporaryFolder;
+import org.jvnet.hudson.test.Issue;
 
-/**
- * @author Kohsuke Kawaguchi
- */
-public class ReopenableRotatingFileOutputStreamTest {
+public class RewindableRotatingFileOutputStreamTest {
```

Comment:
`ReopenableRotatingFileOutputStream` was deprecated in #2490 and `SlaveComputer` now uses `RewindableRotatingFileOutputStream`, so test that.